firedrake/preconditioners/facet_split.py

```diff
@@ -143,11 +144,10 @@ def _restrict_nullspace(nsp):
 
     def update(self, pc):
         if hasattr(self, "_permute_op"):
-            for mat in self.pc.getOperators():
-                mat.destroy()
```

Comment:
Will these now just get cleaned up automatically by garbage collection?

Reply:
The operators are now the same petsc objects, so we should not destroy them.